client/src/views/Home/index.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState, useEffect } from "react";
+import MovieItem from "./MovieItems";
+import SearchBar from "../../components/SearchBar";
+import HomeText from "../../components/HomeText";
+import SearchError from "../../components/SearchError";
+import Loader from "../../components/Loader";
+import Illustration from "../../images/2.Illustrations/illustration-empty-state.png";
+import "./styles.scss";
+import axios from "axios";
+
+const Home = ({ favorites }) => {
+  const [searchText, setSearchText] = useState("");
+  const [moviesList, setMoviesList] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [searchError, setSearchError] = useState(false);
+
+  useEffect(() => {
+    if (searchText) {
+      const httpRequest = setTimeout(async () => {
+        setIsLoading(true);
+        const { data } = await axios.get(
+          `http://www.omdbapi.com/?s=${searchText}&apikey=${process.env.REACT_APP_OMDB_API_KEY}`,
+        );
+        console.log("http request");
```

Comment:
Use console.log only for development purpose, do not deliver an application throwing consoles in production.

Reply:
Removed logs